pool/entities.go

```diff
@@ -0,0 +1,256 @@
+package pool
+
+type ID string
+
+type Name string
+
+// I don't know i have to bring Peer Available storage and other metric inside Peer
+type Peer string
```

Comment:
What does this time represent?

Reply:
I did not get you. If you pointing to the Peer for simplicity I think of it as a PeerID.